py/images/demo.py

```diff
@@ -63,15 +63,15 @@ def ansible_bind_mount(self, target):
             run(['umount', str(setup_path)])
             setup_path.rmdir()
 
-    def setup_ansible(self, target, domain):
+    def setup_ansible(self, target, config_yml):
         if not Path('/usr/bin/ansible-playbook').is_file():
```

Comment:
Not related to the PR, but I think this would be more robust as `if shutil.which('ansible-playbook'): ...`.
Docs for [shutil.which](https://docs.python.org/3.3/library/shutil.html#shutil.which)

Reply:
Ah, that's useful. Do you want me to change it?